commonly_proposed.md

```diff
@@ -37,6 +37,8 @@ Several of the discussions below refer to "C family" languages. This is intended
  * [Use pattern-matching in `if let` instead of optional-unwrapping](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20160201/008979.html): We actually tried this and got a lot of negative feedback, for several reasons: (1) Most developers don't think about things in "pattern matching" terms, they think about "destructuring". (2) The vastly most common use case for `if let` is actually for optional matching, and this change made the common case more awkward. (3) This change increases the learning curve of Swift, changing pattern matching from being a concept that can be learned late to something that must be confronted early. (4) The current design of `if case` unifies "pattern matching" around the `case` keyword.  (5) If a developer unfamiliar with `if case` runs into one in some code, they can successfully search for it in a search engine or Stack Overflow.
 
  * [Syntactic sugar for `if let` self-assignment](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20160829/026796.html): An alternative syntax (such as `if let foo? { ... }` or `if let foo=? { ... }`) to serve as a shorthand for `if let foo = foo { ... }` is often proposed and rejected because it is favoring terseness over clarity by introducing new magic syntactic sugar.
+ 
+ * [Remove or deprecate the force-unwrap operator `!`](https://forums.swift.org/t/moving-toward-deprecating-force-unwrap-from-swift/43455/82): Deprecating the force-unwrap operator (or `try!`) is out of the question, even in a mode enabled via a compiler flag. This is not just for source stability reasons: force-unwrap and force-try are legitimately useful parts of the language.
```

Comment:
My concern is that the phrasing "out of the question" feels overly antagonistic.  How about inverting the text to make it sound more constructive?

> The force-unwrap and force-try remain useful parts of the language used by many people.  For this reason, as well as the pragmatic requirement to maintain source stability (of which these operations factor in prevalently), the core team believes these should remain as part of the language indefinitely.

Reply:
The core team feedback specifically stated that it's "out of the question" and asked for "some wording to this effect" in this document, but we can certainly soften it :)